src/ripple/protocol/TxFlags.h

```diff
@@ -147,15 +147,15 @@ constexpr std::uint32_t const tfNFTokenMintMask =
     ~(tfUniversal | tfBurnable | tfOnlyXRP | tfTransferable);
 
 // NFTokenCreateOffer flags:
-constexpr std::uint32_t const tfSellNFToken            = 0x00000001;
+constexpr std::uint32_t const tfSellNFToken                = 0x00000001;
 constexpr std::uint32_t const tfNFTokenCreateOfferMask =
```

Comment:
It seems the equals signs in this file are aligned with the one on this line because it has the longest prefix. Two things:

1. After changing "NFT" to "NFToken", the new longest prefix is on line 85. The equals sign on that line should be one space apart from that prefix. In this commit, it is four spaces apart.
2. The equals sign on this line, 151, should be aligned with the others in the file.

Reply:
happy to go with that spacing guidance, i had assumed it was a nearest every 4 spaces approach since the last one happened to be % 4 in a clean way

Reply:
@thejohnfreeman for point 2. I think I have seen a different approach in other lines that fold so I thought it was more consistent to not align when they fold, see lines https://github.com/XRPLF/rippled/blob/develop/src/ripple/protocol/TxFlags.h#L67 for example

Reply:
Out of curiosity, has it ever been considered to have formatting rules controlled by a linting tool automatically?

Reply:
The [file](https://github.com/XRPLF/rippled/blob/db3b4dd396c210a90c9e29e2d1ea7046b339460e/src/ripple/protocol/TxFlags.h) as it looked before clang-format hit it in the Great Reformat appears to have every equals sign aligned, even on lines that wrap. They are all in column 44, and they are all 4 spaces after the longest prefix. Both of these rules match the alignment you have, @kennyzlei, except for the wrapped lines that you are not touching. Let's keep that alignment and just add a comment at the start, before or after the `// clang-format off` line, explaining this rule. Optionally, you can restore the wrapped lines to conform to the rule.

@justinr1234 We do have that. It's controlled by clang-format. Style configuration in [.clang-format](https://github.com/XRPLF/rippled/blob/develop/.clang-format). Enforcement in [.github/workflows/clang-format.yml](https://github.com/XRPLF/rippled/blob/develop/.github/workflows/clang-format.yml). We disable it for sections of special formatting like this. When deciding to incorporate it, the majority wanted to keep the special formatting.

Reply:
@thejohnfreeman added a comment to describe the special formatting